dotnet/src/Plugins/Plugins.Web/Bing/BingTextSearch.cs

```diff
@@ -256,7 +256,7 @@ private static string BuildQuery(string query, TextSearchOptions searchOptions)
                     }
                     else
                     {
-                        throw new ArgumentException($"Unknown equality filter clause field name, must be one of {string.Join(",", s_queryParameters)}", nameof(searchOptions));
+                        throw new ArgumentException($"Unknown equality filter clause field name '{equalityFilterClause.FieldName}', must be one of {string.Join(",", s_queryParameters)}", nameof(searchOptions));
```

Comment:
nit. (space after comma)
```suggestion
                        throw new ArgumentException($"Unknown equality filter clause field name '{equalityFilterClause.FieldName}', must be one of {string.Join(", ", s_queryParameters)}", nameof(searchOptions));
```